net-proxy/trojan/trojan-1.8.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils python-utils-r1
```

Comment:
Inheriting utility eclass doesn't do anything. You're looking for the eclass for build-time dependencies. As described in detail on the linked page, and the eclass manual linked to it.